testnets/_IBC/archwaytestnet-akashtestnet.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "archwaytestnet",
```

Comment:
wrong order, akashtestnet should be chain1, and archwaytestnet as chain2

Reply:
Ok, I wonder why your workflow did not catch this one?

Reply:
Turns out the workflow only validated mainnet connections, so now I'm just about to add it for testnet connections, too